src/Classes/iTones/iTones_Playlist.php

```diff
@@ -235,6 +235,27 @@ public function getTitle()
         return $this->title;
     }
 
+    public function getPlaylistAtTime($time)
+    {
+        $getplaylist = self::$db->fetchColumn(
+        'SELECT playlist.playlistid
+        FROM jukebox.playlists playlist
+        INNER JOIN jukebox.playlist_availability avail ON playlist.playlistid=avail.playlistid
+        WHERE playlist.category=2
```

Comment:
Do we have a `Config` variable for the category ID? We'd rather not hard-code it if possible.